docs/debugging.md

```diff
@@ -56,3 +56,9 @@ You *can't*:
 * Perform most other ordinary debugging scenarios.
 
 Development of further debugging scenarios is an on-going focus of the engineering team.
+
+## Troubleshooting tips
+
+If you are running into errors, the following tip may help.
```

Comment:
* Use contraction: you are :point_right: you're
* Change the period at the end to a colon.